### ISSUE TITLE
Away Missions

### ISSUE THREAD

**Issue by LZakida** [addictgamer/Terrastation-13#33]
Ok, currently we have three away missions on the possible list out of a huge number of them. I recently removed Beach from the list because it's boring as butts.

But(t) this issue has two purposes:

1) discussion about what of the existing away missions should be included
2) brainstorming about new ones we can implement for OUR code

To my current recollection, there are a few randomZ that I would like to see become possibilities BUT they aren't currently, possibly because they are borken. As for us, I kinda wanted something horrifying to be a possible option but be something WE add, and not something we took from another.

**Comment by Amauros** [comment]
If for example, an alien infested world were on the other side of the gate and the gate was active... could alien NPC's wander through the gate into the station?

**Comment by taukausanake** [comment]
Here's something that was suggested on Paradise but never got any love. A mirror of the current station but bloodied the fuck up. Maybe nukes or shadowlings or something and there's nothing but gibs everywhere. Maybe some special rewards and some dangerous creatures.

Also there are enough away missions that Paradise has loaded. Beach is a good one so I say keep it in. Why not just load them all in?

**Comment by LZakida** [comment]
@Amauros 
Yes, in theory. To my knowledge all the current maps are built in such a way to prevent that very thing from happening.
@taukausanake 
About the mirror level, I love that idea. 
About your second paragraph:
1) I think that a bunch of the away missions in question are broken. I might be and probably am wrong about that though; I haven't checked at all recently.
2) Beach? Good? there is literally nothing there except beachy stuff. I know that might appeal to some people but honetly I want to make the away missions 3spoopy5everyone lol
